cli/src/pixl_cli/_docker_commands.py

```diff
@@ -42,11 +43,17 @@ def dc(args: tuple[str]) -> None:
 
 
 def _parse_up_args(args: tuple[str, ...]) -> list:
-    """Check up args and define default profile if not set"""
+    """Check up args and set docker compose profile"""
     args_list = list(args)
-    if "--profile" not in args:
-        up_index = args.index("up")
-        args_list[up_index:up_index] = ["--profile", "postgres-exposed"]
+
+    up_index = args.index("up")
+    external_pixl_db_env = os.environ.get("EXTERNAL_PIXL_DB", "false").lower()
```

Comment:
I think we're using decouple's config elsewhere for this so probably worth using that for consistency 😄

Reply:
Ah, orthanc-anon and orthanc-raw plugins use os.environ.get() but I'll amend to decouple, which will be consistent within pixl_cli 👍

Reply:
Oh interesting that they do, we should probably also change that but that doesn't need to happen this PR